### DISCUSSION TITLE
How to uninstall

### DISCUSSION THREAD

**Discussion by sth9634** [ichi4096/vivado-on-silicon-mac#61] [Q&A] [answered]
Thanks for great software. 
As I got windows pc, I want to uninstall vivado on my Mac.

Can I just delete whole folder for it?

**Comment by ichi4096** [comment, marked as answer]
Yes, deleting the folder will uninstall it since the Vivado installation is contained wholly within the folder. Remember to remove it from the trash as well to save disk space.

Glad my scripts worked for You until then.

**Reply by sth9634** [reply]
Thanks a lot!
